pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -84,6 +84,7 @@ const (
 	MasterRoleName            = "system:master"
 	NodeRoleName              = "system:node"
 	NodeProxierRoleName       = "system:node-proxier"
+	PrivilegedRouterRoleName  = "system:privileged-router"
```

Comment:
How many router flavors do we have?  Seems like we might want to name this specifically for F5, so that if you need to "list privileged-fobbers" next, we don't need any new bindings.